src/services/impl/SanitizerImpl.ts

```diff
@@ -0,0 +1,14 @@
+import {Sanitizer} from "../Sanitizer";
+
+export class SanitizerImpl implements Sanitizer {
+
+  validateEmail(email?: string): string | null {
```

Comment:
If we would like to validate an email, it means we receive one in input. email should not be nullable from my point of view